enclave/core/switchlesscalls.c

```diff
@@ -37,40 +38,28 @@ bool oe_is_switchless_initialized()
 **
 **==============================================================================
 */
-oe_result_t oe_handle_init_switchless(uint64_t arg_in)
+oe_result_t oe_handle_init_switchless(
```

Comment:
Sort of an inherent problem in the original implementation, but it's a little bit more obvious now that this is a function in the publicly inspectable EDL: this initialization is only expected to happen once per enclave initialization from the host internally as part of `oe_create_enclave()`  but there's nothing preventing it from being invoked elsewhere.

Particularly since there are no synchronization guards around the globals this method initializes, the enclave side of this needs to enforce `oe_once()` semantics around them, as happens in `oe_initialize_enclave()`

Reply:
I added synchronization to prevent this here: https://github.com/openenclave/openenclave/pull/2433

@mingweishih can you rebase against master?

Reply:
Thanks @jhand2 for point that out. I'm also aware of that patch (which introduces conflict to this PR). I'll rebase the PR.